src/frontend/layout-template.tsx

```diff
@@ -49,7 +49,9 @@ const html = async (
       <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
       <title>${branding.companyName}</title>
       ${faviconTag}
-
+      <style>
+        #app { isolation: isolate; }
```

Comment:
I think this should be a part of existing global style: https://github.com/SoftwareBrothers/adminjs/blob/master/src/frontend/components/application.tsx#L18

Reply:
fixed